Resources/Textures/_Harmony/Objects/Weapons/Guns/Basic/staves.rsi/meta.json

```
@@ -0,0 +1,22 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Sprites by mithrandalf at https://github.com/ParadiseSS13/Paradise/pull/10557/commits/9111cb1c22407e3bb9098812ea7727fb68726d3fhttps://github.com/ParadiseSS13/Paradise/pull/10557/commits/9111cb1c22407e3bb9098812ea7727fb68726d3f",
```

The source link was written twice
```suggestion
    "copyright": "Sprites by mithrandalf at https://github.com/ParadiseSS13/Paradise/pull/10557/commits/9111cb1c22407e3bb9098812ea7727fb68726d3f",
```

---

I also prefer merge commit or commit as a source, not pull request or commit in a pull request. PRs can disappear, commits cannot 

https://github.com/ParadiseSS13/Paradise/commit/a1f9a19fe2e44523cba8cc05373fc440ab571590